<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Klient Tumblr</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>favicon.gif</normaloff>favicon.gif</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>351</width>
      <height>421</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_regular">
     <attribute name="title">
      <string>tekst</string>
     </attribute>
     <widget class="QLineEdit" name="regular_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>tytuł</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>treść</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="regular_body">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_quote">
     <attribute name="title">
      <string>cytat</string>
     </attribute>
     <widget class="QPlainTextEdit" name="quote_quote">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="quote_source">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>301</width>
        <height>171</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>cytat</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>źródło</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_conversation">
     <attribute name="title">
      <string>rozmowa</string>
     </attribute>
     <widget class="QLineEdit" name="conversation_title">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="conversation_conversation">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>tytuł</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>rozmowa</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_url">
     <attribute name="title">
      <string>odnośnik</string>
     </attribute>
     <widget class="QLineEdit" name="link_name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="link_url">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="link_description">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>301</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>nazwa</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>opis</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_photo">
     <attribute name="title">
      <string>zdjęcie</string>
     </attribute>
     <widget class="QToolButton" name="photo_path_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="photo_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="photo_caption">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>plik</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>opis</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_video">
     <attribute name="title">
      <string>film</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>plik</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>opis</string>
      </property>
     </widget>
     <widget class="QToolButton" name="video_path_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="video_caption">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="video_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_audio">
     <attribute name="title">
      <string>dźwięk</string>
     </attribute>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>plik</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>64</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>opis</string>
      </property>
     </widget>
     <widget class="QToolButton" name="audio_path_button">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>26</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="audio_caption">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>301</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="audio_path">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>261</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="publish">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Publikuj</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>regular_title</tabstop>
  <tabstop>regular_body</tabstop>
  <tabstop>publish</tabstop>
  <tabstop>tabs</tabstop>
  <tabstop>quote_quote</tabstop>
  <tabstop>quote_source</tabstop>
  <tabstop>conversation_title</tabstop>
  <tabstop>conversation_conversation</tabstop>
  <tabstop>link_name</tabstop>
  <tabstop>link_url</tabstop>
  <tabstop>link_description</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
